safetab_p/TESTPLAN.md

```diff
@@ -28,22 +28,24 @@ python3 -m nose $DIR/safetab_utils -a '!slow'
 *Slow Tests:*
 
 ```bash
-DIR=<path of cloned repository>
-python3 -m nose $DIR/common -a 'slow'
-python3 -m pytest $DIR/analytics -m 'slow'
+DIR=<absolute path of cloned DAS_2020_DDHCA_Production_Code repository>
+python3 -m nose $DIR/tumult/common -a 'slow'
+python3 -m pytest $DIR/tumult/analytics -m 'slow'
 python3 -m nose $DIR/safetab_p -a 'slow'
 python3 -m nose $DIR/safetab_utils -a 'slow'
 (Total runtime estimate: 140 minutes)
 ```
 
 ## SafeTab-P's tests
 
+*All tests are designed to be run from the directory `<absolute path of cloned DAS_2020_DDHCA_Production_Code repository>/safetab_p/tmlt/safetab_p`.*
```

Comment:
That's correct. I think that this was a slight typo. I fixed things and reran all of the commands to ensure that they worked. Resolving this but feel free to re-open on the new line if you're unhappy.

Reply:
I would prefer to have the test command paths relative to the repository root, rather than the `safetab_p` folder. That's consistent with the rest of the documentation.